src/field_theory/adjoin.lean

```diff
@@ -117,33 +121,125 @@ begin
             (subset_adjoin _ _) },
 end
 
-variables (α : E)
+/-- `F[S][T] = F[T][S]` -/
+lemma adjoin_adjoin_comm (T : set E) :
+  (adjoin (adjoin F S : set E) T : set E) = (adjoin (adjoin F T : set E) S : set E) :=
+by rw [adjoin_adjoin_left, adjoin_adjoin_left, set.union_comm]
 
-notation K`⟮`:std.prec.max_plus l:(foldr `, ` (h t, set.insert h t) ∅) `⟯` := adjoin K l
+/--
+Variation on `set.insert` to enable good notation for adjoining elements to fields.
+Used to preferentially use `singleton` rather than `insert` when adjoining one element.
+-/
+--this definition of notation is courtesy of Kyle Miller on zulip
+class fancy_insert {α : Type*} (s : set α) :=
```

Comment:
We're inside the `field` namespace right? In that case I would drop the `fancy_` prefix.